### DISCUSSION TITLE
Encoding of non-english search query

### DISCUSSION THREAD

**Discussion by yongbin** [kagisearch/kagi-docs#527] [Kagi Search API Feature Requests & Ideas]
I have a question about using the Search API for non-English languages. 

I checked the example for 'microsoft' in the enrich api example and called it to get the result. However, I did not succeed when I changed the query to my native language (Korean). I tried two ways to pass the UTF-8 encoded q directly and base64 encoded, but it was the same. How should I pass the query to use a non-English language as a query in the API? 

```sh
# Success
curl -v \
  -H "Authorization: Bot XXX" \
  "https://kagi.com/api/v0/enrich/news?q=7Lm07Lm07Jik7Y6Y7J207IaQ7ZW067O07ZeY"

# Fail
curl -v \
  -H "Authorization: Bot XXX" \
  "https://kagi.com/api/v0/enrich/news?q=삼성전자"

# Fail
curl -v \
  -H "Authorization: Bot XXX" \
  "https://kagi.com/api/v0/enrich/news?q=7Lm07Lm07Jik7Y6Y7J207IaQ7ZW067O07ZeY"

```

**Comment by patchoulish** [comment]
You'll want to percent-encode the query string you append to your URL, i.e.
```
https://kagi.com/api/v0/enrich/news?q=%EC%82%BC%EC%84%B1%EC%A0%84%EC%9E%90
```
